### ISSUE TITLE
shadow 2.12.2 seems to not be able to compile reagent

### ISSUE THREAD

**Issue by Wegi** [thheller/shadow-cljs#866]
Hi, 

with the new version we get a compile error, when compiling re-frame.

```clojure
Multiple files failed to compile.
aborted par-compile, [:shadow.build.classpath/resource "reagent/impl/component.cljs"] still waiting for #{react}
{:aborted [:shadow.build.classpath/resource "reagent/impl/component.cljs"], :pending #{react}}
ExceptionInfo: aborted par-compile, [:shadow.build.classpath/resource "reagent/impl/component.cljs"] still waiting for #{react}
	shadow.build.compiler/par-compile-one (compiler.clj:1007)
	shadow.build.compiler/par-compile-one (compiler.clj:972)
	shadow.build.compiler/par-compile-cljs-sources/fn--14236/iter--14258--14262/fn--14263/fn--14264/fn--14265 (compiler.clj:1090)
	clojure.core/apply (core.clj:667)
	clojure.core/with-bindings* (core.clj:1977)
	clojure.core/with-bindings* (core.clj:1977)
	clojure.core/apply (core.clj:671)
	clojure.core/bound-fn*/fn--5767 (core.clj:2007)
	java.util.concurrent.FutureTask.run (FutureTask.java:264)
	java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1130)
	java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:630)
	java.lang.Thread.run (Thread.java:832)
```

Thanks for the awesome library otherwise. :)

**Comment by thheller** [comment]
I just tested with a fresh `lein new re-frame rf-test` and it compiles fine?

This is with a `:browser` build?

**Comment by Wegi** [comment]
Yes, I will test further and try to find the root. Downgrading to 2.12.1 does not cause the error though.

**Comment by Wegi** [comment]
Hey, we get the build to work with `:browser` target as well. But it seems the `:node-test` target is the problem.

**Comment by nathell** [comment]
Can confirm it on our end. We are consistently seeing builds with `:node-test` target failing on 2.12.2 and working on 2.12.1.

**Comment by thheller** [comment]
Should be fixed as of `2.12.3`.
